audio_separator/separator/separator.py

```
@@ -654,13 +941,19 @@ def load_model_data_from_yaml(self, yaml_config_filename):
+            )
         else:
             model_data_yaml_filepath = yaml_config_filename
 
-        self.logger.debug(f"Loading model data from YAML at path {model_data_yaml_filepath}")
+        self.logger.debug(
+            f"Loading model data from YAML at path {model_data_yaml_filepath}"
+        )
 
-        model_data = yaml.load(open(model_data_yaml_filepath, encoding="utf-8"), Loader=yaml.FullLoader)
+        model_data = yaml.load(
+            open(model_data_yaml_filepath, encoding="utf-8"), Loader=yaml.FullLoader
```

Ruff (0.15.6)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)